iroh-net/src/hp/netcheck/probe.rs

```diff
@@ -94,9 +109,39 @@ impl Probe {
 /// A set of probes is done once either one of the probes completes, or
 /// the next probe to run wouldn't yield any new information not
 /// already discovered by any previous probe in any set.
+///
+/// Generated probe plans have the following shape:
+///
+/// - Each probe **set** is single identical probe, but multiple tries with delayed starts.
```

Comment:
hi hi! In a few places, this included, the conversation is marked as resolved, but the suggestion wasn't taken and there isn't a response as to why. Were these skipped over?

Confused specifically in this case, because there is a grammar error here that should be fixed regardless of phrasing.

Reply:
oh yeah, that's weird now you point it out.  I guess it should show it as outdated.  I promise I don't (intentionally) dismiss reviews without acting on them!  If I click "resolved" I mean I did do the obvious requested fix or something very close to it that I expect to be unobjectionable (perhaps not 100% literally like in this case my editor re-flowed the paragraph).